sample-apps/spark/build.gradle.kts

```diff
@@ -41,3 +51,15 @@ tasks {
     dependsOn(":otelagent:jibDockerBuild")
   }
 }
+
+tasks.register<com.google.cloud.tools.jib.gradle.BuildImageTask>("jibBuildWithoutAgent") {
```

Comment:
please use imports instead of the full coordinates to the `BuildImageTask` and `JibExtension`.

you can also put this code inside the tasks block.

Reply:
@rapphil thanks. updated it accordingly.